### ISSUE TITLE
Docs self hosting 

### ISSUE THREAD

**Issue by RenameMe1** [strawberry-graphql/strawberry#3573]
Hello, Tell me please, is it possible to self-host post the documentation the way it is done in FASTAPI (https://fastapi.tiangolo.com/how-to/custom-docs-ui-assets/?h=) ? 

I can also give an example of replacing “make_graphiql_handler from starlette_graphene3” with “HTMLResponse(_CUSTOM_GRAPHIQL_HTML)”, where all URLs on the path to the files are changed to StaticFiles (https://fastapi.tiangolo.com/reference/staticfiles/?h=stat)

Our development environment is isolated, so access to "https://unpkg.com/*" is impossible.

**Comment by patrick91** [comment]
@RenameMe1 you can do this already, but we don't provide anything for that unfortunately, the implementation would depend on the framework you use.

I think we could add this if we had a nice way to integrate it with all frameworks

**Comment by RenameMe1** [comment]
@patrick91 At the moment is it to change the URLs in https://github.com/strawberry-graphql/strawberry/blob/main/strawberry/static/graphiql.html?

**Comment by patrick91** [comment]
@RenameMe1 yes, or implement a view that returns the html in your framework of choice 😊

maybe a middleground would be us writing some guides on how to do it 😊

**Comment by RenameMe1** [comment]
@patrick91 
Thanks for the help. I will try
